builder/README.md

```diff
@@ -1,6 +1,9 @@
 # Deis Builder
 
-A Docker image that builds Docker images, for use in the [Deis](http://deis.io/) open source PaaS.
+**WARNING:** This is the experimental Go-based builder.
```

Comment:
Should the **WARNING** line be replaced with the old one above it before merge? Or did we want to leave it in there until closer to the next release?